src/components/Navigation/SideBar/MenuItem.styles.ts

```diff
@@ -0,0 +1,63 @@
+import { Typography } from '@equinor/eds-core-react';
+import { tokens } from '@equinor/eds-tokens';
+
+import { spacings } from 'src/style';
+
+import styled from 'styled-components';
+
+const { colors } = tokens;
+
+interface LinkProps {
+  $active?: boolean;
+  $open?: boolean;
+  $disabled?: boolean;
+}
+
+export const Link = styled.a<LinkProps>`
```

Comment:
Nitpicky: Can we use [Link](https://reactrouter.com/en/main/components/link) from `react-router-dom` instead here? That way we avoid reloads of the page

Reply:
Done